.github/ISSUE_TEMPLATE/bug_shownotes.yml

```diff
@@ -0,0 +1,47 @@
+name: Show Errors
+description: This type of issue is for problems with the episode itself (links, sponsors, typos, etc)
+labels:
+  - bug
+  - show notes
+  - "piece of :cake:"
+  - good first issue
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ### Please check to see if someone else has already reported this bug!
```

Comment:
I like this reminder 😁, but is it possible to link to the issues on the word bug? Just so people know what we mean by that if they don't use github very much.

Also, I think the `!` might get interpreted the wrong way since the message will already be bolded with the header text. Do you think a period would be enough? (If we change it here, then we should change it for the other templates as well (just the areas reminding them to check for previous issues)) 

I know these are small detail, but I'm just trying to think of all the ways it could be mis-understood/interpreted 😅